src/index.js

```diff
@@ -376,7 +376,10 @@ function ncc (
             loader: eval('__dirname + "/loaders/shebang-loader.js"')
           }]
         }
-      ]
+      ],
+      parser: {
+        javascript: { importMeta: false },
+      },
```

Comment:
Since ncc supports both CJS and ESM modes, I think we need to make this conditional based on the entrypoint.

Reply:
Right, this certainly isn't necessary in CJS modules (though I guess leaving it in the config then wouldn't matter). Thinking about this, what if an ESM dependency of a CJS module uses `import.meta`? I guess the current config would also affect the ESM module. Anyway, I'm about to add some tests for this!

Reply:
Hmm good point! I tried `console.log(import.meta.url)` in a cjs file and node gave me an error

> SyntaxError: Cannot use 'import.meta' outside a module

So should be fine to always enable, as long as this is the only behavior thats changed.

Speaking of which, I can't seem to find  any documentation on `importMeta`. Did you mean `importMetaContext`?

https://webpack.js.org/configuration/module/

Reply:
> Speaking of which, I can't seem to find any documentation on importMeta. Did you mean importMetaContext?

`importMeta` is just right – it looks like it didn't yet make it into the docs; it's in the config types though.

Reply:
there now, https://webpack.js.org/configuration/module/#moduleparserjavascriptimportmeta